low-level/solution/lr2/server/health.py

```diff
@@ -82,6 +82,18 @@ def __init__(self):
         self._ipmi = IpmiFactory().get_implementor("ipmitool")
         self.platform_sensor_list = ['Temperature', 'Voltage', 'Current']
         self.service = Service()
+        # Resources and their common key path.
+        self.manifest_health_index_map_key = {
```

Comment:
rename this to resource_indexing_map{} and maintain in respective server / storage resource map class

Reply:
Done